virttest/utils_test/libvirt.py

```diff
@@ -3577,3 +3578,14 @@ def get_disk_alias(vm, source_file=None):
                 logging.info("Ignore error of source attr getting for file: %s" % e)
                 pass
     return None
+
+
+def check_cmd_expected(check_cmd, expected_match, is_expected):
```

Comment:
Function explanation and parameters explanation.

Reply:
And you can consider more scenario, like not expected match

Reply:
thks for advice. Add commend and add a new scenario